### DISCUSSION TITLE
adding tax to filter for list view

### DISCUSSION THREAD

**Discussion by Gazzxy** [YetiForceCompany/YetiForceCRM#15348] [How to] [answered]
Would like to have tax summary on a filter in list view.. for sales invoices. however I cannot see it in the list at all for available fields.  

only thing I seen similar was someone else struggling with it but the threat amounts to I had this problem.. till I didn't, and I cannot really figure out what he was on about with his fix.

**Comment by rskrzypczak** [comment, marked as answer]
Advanced block fields are not available in list view.
If you want them to be visible, you need to create new fields, as is the case with Gross, Net field.
updating the data in these fields can take place automatically if you name them appropriately, as mentioned in this [issue](https://github.com/YetiForceCompany/YetiForceCRM/issues/12571#issuecomment-586957074).
However, be aware that the data will not complete itself for existing entries.

**Comment by Gazzxy** [comment]
thanks that makes sense. 

and bummer.. thankfully not tooo many record to go and fill that field in
